From 328380d53f4af4daede53839ed81a83fc678fb7e Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 3 Aug 2020 12:12:35 +0800
Subject: [PATCH] Test case2 for Add a new include inc file

Description: Add a new include inc file ("!include TestPkgDefines.fdf.inc") in TestPkg/TestPkg32X64.dsc.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/TestPkg32X64.dsc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/TestPkg/TestPkg32X64.dsc b/TestPkg/TestPkg32X64.dsc
index 532d084223..c93966250d 100644
--- a/TestPkg/TestPkg32X64.dsc
+++ b/TestPkg/TestPkg32X64.dsc
@@ -43,6 +43,7 @@
   DEFINE NETWORK_ALLOW_HTTP_CONNECTIONS = TRUE
 
 !include NetworkPkg/NetworkDefines.dsc.inc
+!include TestPkgDefines.fdf.inc
 
   #
   # Device drivers
-- 
2.27.0.windows.1

